[CI] XPackRestIT test {p0=analytics/nested_top_metrics_sort/terms order by top metrics numeric not null double values} failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #10739 / almalinux-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/yzobod3skghnu)
- [elasticsearch-pull-request #93712 / rest-compatibility](https://gradle-enterprise.elastic.co/s/r7uvnimongugk)
- [elasticsearch-pull-request #93657 / part-3](https://gradle-enterprise.elastic.co/s/2ebhpimbtgls4)
- [elasticsearch-pull-request #93302 / part-3](https://gradle-enterprise.elastic.co/s/xclcbhmhthkqk)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:yamlRestTest" --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {p0=analytics/nested_top_metrics_sort/terms order by top metrics numeric not null double values}" -Dtests.seed=1B094431A7C75FA2 -Dtests.locale=ta -Dtests.timezone=Etc/GMT+4 -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.test.rest.XPackRestIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bp0%3Danalytics%2Fnested_top_metrics_sort%2Fterms%20order%20by%20top%20metrics%20numeric%20not%20null%20double%20values%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [analytics/nested_top_metrics_sort:101]: got unexpected warning header [
	299 Elasticsearch-9.2.0-a3f5ea5a6b58d225499eb80c6eb15dd97d0c0964 "Index [.ml-annotations-000001] name begins with a dot (.), which is deprecated, and will not be allowed in a future Elasticsearch version."
]

```

**Issue Reasons:**
- [main] 4 failures in test test {p0=analytics/nested_top_metrics_sort/terms order by top metrics numeric not null double values} (0.4% fail rate in 936 executions)
- [main] 2 failures in step part-3 (0.9% fail rate in 225 executions)
- [main] 3 failures in pipeline elasticsearch-pull-request (1.2% fail rate in 244 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -159,15 +159,6 @@ tests:
 - class: org.elasticsearch.aggregations.bucket.AggregationReductionCircuitBreakingIT
   method: testCBTrippingOnReduction
   issue: https://github.com/elastic/elasticsearch/issues/134667
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=transform/transforms_start_stop/Test stop transform with force and wait_for_checkpoint true}
-  issue: https://github.com/elastic/elasticsearch/issues/135135
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=analytics/nested_top_metrics_sort/terms order by top metrics numeric not null double values}
-  issue: https://github.com/elastic/elasticsearch/issues/135159
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=analytics/nested_top_metrics_sort/terms order by top metrics numeric not null integer values}
-  issue: https://github.com/elastic/elasticsearch/issues/135162
 - class: org.elasticsearch.gradle.TestClustersPluginFuncTest
   method: override jdk usage via ES_JAVA_HOME for known jdk os incompatibilities
   issue: https://github.com/elastic/elasticsearch/issues/135413

